### PR TITLE
Handle exceptions raised from validate output function

### PR DESCRIPTION
**Pull request by xhaakon**
An exception raised from e.g. dab_response_validator now stops execution of the test suite. Instead, mark the case as a failure and move on.

### CODE REVIEW

**Comment by xhaakon** [comment]
Looks like this just a more universal solution to https://github.com/device-automation-bus/dab-compliance-suite/pull/25. I'll leave it to the maintainers to pick whichever pull request they prefer.

**Review by bgy36ww — APPROVED** [review]

**Pull request merged**
